azext_iot/sdk/digitaltwins/dataplane/operations/digital_twins_operations.py

```diff
@@ -947,7 +953,7 @@ def internal_paging(next_link=None, raw=False):
     list_incoming_relationships.metadata = {'url': '/digitaltwins/{id}/incomingrelationships'}
 
     def send_telemetry(
-            self, telemetry, id, message_id, telemetry_source_time=None, digital_twins_send_telemetry_options=None, custom_headers=None, raw=False, **operation_config):
+            self, id, telemetry, message_id, telemetry_source_time=None, digital_twins_send_telemetry_options=None, custom_headers=None, raw=False, **operation_config):
```

Comment:
telemetry_source_time and digital_twins_send_telemetry_options need to be integrated for the corresponding command here and for send_component_telemetry (and dt_timestamp removed). Looks like it was missed in a prior SDK refresh (the original parameter was dt_timestamp).

Reply:
The user facing param could be something like:

`--telemetry-source-time`: `An RFC 3339 timestamp that identifies the time the telemetry was measured. If not provided the client will generate a UTC timestamp of when the request was processed.`

Reply:
done - also made it so that the client will not generate the timestamp. That way the service can decide what to do if there is no telemetry source time (like how it was doing that before)